src/stitch_fix/generateId.js

```diff
@@ -1,6 +1,8 @@
 import { ulid } from 'ulid';
 import { constantCase } from 'constant-case';
 
+// Keep the type signature of this function in sync with the types exposed in
+// index.d.ts
 const generateId = (key) => {
   if (typeof key === 'undefined') {
```

Comment:
the nice thing is that a TS app now won't let the `key` be `undefined` thanks to these type definitions.